### PR TITLE
reverted/added back seekbyte into the 'base' hf_sim_params for easier…

### PR DESCRIPTION
**Pull request by shot0829**
… track. also added back a line of missing params from last merge

#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

File: scripts/hf_sim.py [hunk 1]
```diff
@@ -400,19 +400,20 @@ def run_hf(local_statfile, n_stat, idx_0, velocity_model=args.velocity_model):
         str(args.vs_moho),
         "%d %s %s %s %s %d" % (nl_skip, vp_sig, vsh_sig, rho_sig, qs_sig, ic_flag),
         velocity_name,
+        "%s %s %s" % (args.fa_sig1, args.fa_sig2, args.rv_sig1),
```
Comment (jasonmotha) [inline scripts/hf_sim.py]: Doesn't this screw up for previous versions of HF eg. 5.4.5.4

Comment (shot0829) [reply scripts/hf_sim.py]: well, this is actually what the previous version needed, this line was somehow skipped from the last PR #294 
took me a while to find what was missing.
And this is why i want to keep the original 'template', so in the future if more params were needed, the base will not change.

File: scripts/hf_sim.py [hunk 2]
```diff
@@ -402,18 +402,22 @@ def run_hf(local_statfile, n_stat, idx_0, velocity_model=args.velocity_model):
         velocity_name,
         "%s %s %s" % (args.fa_sig1, args.fa_sig2, args.rv_sig1),
         str(args.path_dur),
-        str(head_total + idx_0 * (nt * N_COMP * FLOAT_SIZE)),
         "",
     ]
+    
+    # extra params needed for v6.0
     if utils.compare_versions(args.version, "6.0.3") >= 0:
         hf_sim_args.insert(
```
Comment (jasonmotha) [inline scripts/hf_sim.py]: Better to change this back to append?

File: scripts/hf_sim.py [hunk 3]
```diff
@@ -402,18 +402,22 @@ def run_hf(local_statfile, n_stat, idx_0, velocity_model=args.velocity_model):
...
+            -1,
             "{} {} {}".format(
                 args.stress_param_adj[0],
                 args.stress_param_adj[1],
                 args.stress_param_adj[2],
             ),
         )
+    # add seekbyte for qcore adjusted version
+    if bin_mod:
+        hf_sim_args.insert(-1, str(head_total + idx_0 * (nt * N_COMP * FLOAT_SIZE)))
```
Comment (jasonmotha) [inline scripts/hf_sim.py]: Same here?

File: scripts/hf_sim.py (hunk 2 above)
Comment (shot0829) [reply scripts/hf_sim.py]: it needs the "' at the end of the list, for a extra newline. 
so unless we removed it from template and then manually add in '\n', we need to insert to the -1 position

Comment (jasonmotha) [reply scripts/hf_sim.py]: Ah yeah. Lets just append the "" at the end of the argument building. It makes it less bug prone.

**Pull request merged**